examples/TestAppUwp/ViewModel/Converters.cs

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.MixedReality.WebRTC;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+
+namespace TestAppUwp
+{
+    /// <summary>
+    /// Convert <c>true</c> to <c>false</c>, and anything else (including <c>false</c>) to <c>true</c>.
```

Comment:
nit: do we expect this to get values that aren't `true` or `false`? Should we throw if that's not the case?

Reply:
For UI I think it's better to try and be as silent as possible rather than throw here and break the UI. I think throwing here also crashes the app since it's called via runtime reflection by the XAML implementation.

Reply:
Yeah, I find the asymmetric behavior a bit weird, but it doesn't matter too much if this is only going to be only used by XAML machinery.